lib/frozen_record/base.rb

```diff
@@ -154,6 +156,13 @@ def add_index(attribute, unique: false)
         self.index_definitions = index_definitions.merge(index.attribute => index).freeze
       end
 
+      def attribute(attribute, klass, deserializer_method: :new)
```

Comment:
I don't really like that `deserializer_method` thing. I think we should either always use `new` or `load` and if people want to use somthing that doesn't respond to the right interface, they can just do a small facade.

Reply:
I went with `load` since I think it's likely the attribute classes would also be used with ActiveRecords and ActiveRecord serialization uses `load`.

We could be _very_ clever and do something like `if klass.responds_to(:load) klass.load(value) else klass.new(value)` but I think that would lead to unexpected behaviour for callers.